src/app/cake/cake.class.ts

```diff
@@ -0,0 +1,13 @@
+
+
+export type CakeStatus =  'Available' |'LastUnits' |'OutOfStock';
```

Comment:
This should be an Enumerator.

Since we are using a Single File Responsibility file distribution, this Enumerator should live in a separate file named `cake-status.enum.ts`.

Reply:
When to use one or the other? They almost accomplish the same thing for this. I know the enum can have certain functionality included, but in this case we are using them just as a constant value.